src/freertos_drivers/ti/CC32xxSPI.cxx

```diff
@@ -154,102 +174,127 @@ void CC32xxSPI::config_dma(struct spi_ioc_transfer *msg)
     /* use DMA */
     void *buf;
     uint32_t channel_control_options;
+    unsigned config_index;
```

Comment:
add todo comment to consider performing the first data write immediately and use the DMA only for n-1 words of TX.